ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -10923,7 +10930,7 @@ void HqlCppTranslator::buildXmlSerializeSetValues(BuildCtx & ctx, IHqlExpression
         CHqlBoundExpr boundCurElement;
         cursor->buildIterateLoop(loopctx, boundCurElement, false);
         OwnedHqlExpr curElement = boundCurElement.getTranslatedExpr();
-        buildXmlSerializeScalar(loopctx, curElement, itemName);
+        buildXmlSerializeScalar(loopctx, curElement, itemName, false);
```

Comment:
The fourth parameter is a contentFlag, not a boolean.  Using an enum for the parameter would have caught it.